Pipfile

```diff
@@ -5,7 +5,10 @@ name = "pypi"
 
 [packages]
 "72eb2aa" = {file = "https://github.com/Rapptz/discord.py/archive/rewrite.zip"}
+aiohttp = "*"
 aiodns = "*"
+bs4 = "*"
+html2text = "*"
```

Comment:
You have edited the `Pipfile` directly, and in a way that will break `discord.py`. Please revert your Pipfile and read over the `doc/` folder again.

Reply:
It does pin aiohttp and websockets, but Pipenv doesn't pick up on it correctly if they aren't specified directly in the pipfile. You also can't just use the latest version of aiohttp, it's not compatible.

I make a PR this morning, maybe check that out.

Reply:
Gotcha, & just saw that! I'll fix all this as soon as I get home.